samples/bot-ai-meeting-helper/README.md

```diff
@@ -0,0 +1,202 @@
+
+## Included Features
+* **Graph API:** Utilize graph APIs to retrieve meeting transcriptions and other meeting or user-related details.
+* **Azure Open AI:** Leverages Azure OpenAI for extracting action items and meeting summaries.
+* **Azure Table Storage:** Utilizes Azure Table Storage to store subscribed user information, including conversation IDs for sending meeting details.
+* **Self-Signed Certificate:** Uses Azure self-signed certificates for encryption and decryption to establish the meeting subscription.
+
+## Interaction with app
+
+ ![bot-ai-meeting-helperGif](Images/bot-ai-meeting-helperGif.gif)
+
+## Try it yourself - experience the App in your Microsoft Teams client
+Please find below demo manifest which is deployed on Microsoft Azure and you can try it yourself by uploading the app package (.zip file link below) to your teams and/or as a personal app. (Sideloading must be enabled for your tenant, [see steps here](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)).
+
+**Microsoft Teams Meeting Helper sample app:** [Manifest](/samples/bot-ai-meeting-helper/demo-manifest/bot-ai-meeting-helper.zip)
+
+- To run the demo, you must set up and grant the necessary policy. Please follow the second step mentioned above for instructions on how to do this. 
+
+## Prerequisites
+
+- [Node.js 18.x](https://nodejs.org/download/release/v18.18.2/)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Table Storage](https://learn.microsoft.com/en-us/azure/storage/tables/table-storage-quickstart-portal)
+- [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+- [Azure OpenAI](https://learn.microsoft.com/en-us/azure/ai-services/openai/quickstart?tabs=command-line&pivots=programming-language-studio)
+- [PowerShell](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-macos) if on macOS (`brew install --cask powershell`)
+- [Self-Signed Certificate](https://learn.microsoft.com/en-us/azure/key-vault/certificates/quick-create-portal)
+
+### Create an Azure Open AI service
+- In Azure portal, create a [Azure Open AI service](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource?pivots=web-portal).
+- **Deploy Azure Open AI model:** Deploy the `gpt-35-turbo` model in your created Azure Open AI service for the application to perform translation.
+- Create and collect `AZURE_OPENAI_ENDPOINT`, `AZURE_OPENAI_API_KEY`, `AZURE_OPENAI_DEPLOYMENT_NAME`, and save those value  to update in `.env` file later.
+
+### Create an Azure Table Storage
+- In Azure portal, create a [Azure Table Storage](https://learn.microsoft.com/en-us/azure/storage/tables/table-storage-quickstart-portal).
+- Create and collect `Account_Name`, `Account_Key`, `Table_Name`, and save those value  to update in `.env` file later.
```

Comment:
Do we need to update .env file or .localConfigs ?

Reply:
fixed